compiler/ir/ir.psi2ir/src/org/jetbrains/kotlin/psi2ir/generators/fragments/EvaluatorFragmentInfo.kt

```diff
@@ -22,5 +22,27 @@ import org.jetbrains.kotlin.descriptors.FunctionDescriptor
 class EvaluatorFragmentInfo(
     val classDescriptor: ClassDescriptor,
     val methodDescriptor: FunctionDescriptor,
-    val parameters: List<DeclarationDescriptor>
+    parameterDescriptors: List<DeclarationDescriptor>,
```

Comment:
@udalov OK, so here's a migration plan:
0. this PR lands
1. intellij-community#1839 lands, using `createWithFragmentParameterInfo` on the companion.
2. we change the constructor to the new signature, and the implementation in the companion accordingly.
3. we change the call site in intellij-community to use the constructor instead.

OK?

Reply:
Sounds great, thanks!